main/client/routes.js

```diff
@@ -0,0 +1,18 @@
+var app = angular.module('app');
```

Comment:
Don't use var app = angular.module. Just use angular.module('name').controller(//....

It can cause problems with minification.